src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -202,6 +223,53 @@ public async override Task<WorkspaceEdit> RemapWorkspaceEditAsync(WorkspaceEdit
             return workspaceEdit;
         }
 
+        private bool TryGetDocumentChanges(WorkspaceEdit workspaceEdit, out TextDocumentEdit[] documentEdits, out SumType<CreateFile, RenameFile, DeleteFile>[] documentNonEditChanges)
```

Comment:
Suggest calling this `documentTextEdits` and `documentFileEdits` to better differentiate. `documentNonEditChanges` is a bit confusing IMO

Reply:
Good idea, though per conversation with Taylor I dropped the second commit and filed you guys #26451